web/static/css/style.css

```diff
@@ -0,0 +1,68 @@
+@font-face { font-family: 'emulogic'; src: url("fonts/emulogic.eot"); src: local(""), url("emulogic.woff") format("woff"), url("emulogic.ttf") format("truetype"), url("emulogic.svg") format("svg"); font-style: normal; font-weight: normal; }
```

Comment:
This looks like a compiled file ... Please don't checkin a compiled file. Since the project isn't set up to work with SCSS, please either set it up to work with SCSS properly and only checking the SCSS files or don't checking your SCSS files, and make sure the CSS is compiled to the project standard format.